### PR TITLE
Carroussel news layout

### PR DESCRIPTION
**Pull request by julianecasier**
Styling CarrousselNews (with new movies) therefore modyfying  general layout

### CODE REVIEW

**Review by Dash293 — DISMISSED** [review]

File: frontend/src/components/Footer/style.jsx [hunk 1]
```diff
@@ -1,5 +1,21 @@
 import styled from "styled-components";
 
 export default styled.footer`
-  height: 20vh;
+  background-color: var(--color-bg-primaire);
+  color: var(--color-font-primaire);
+
```
Comment (LuigiMenez) [inline frontend/src/components/Footer/style.jsx]: warning blank line

File: frontend/src/components/Suggerator/style.jsx [hunk 2]
```diff
@@ -2,4 +2,10 @@ import styled from "styled-components";
 
 export default styled.section`
   height: 20vh;
+  max-width: 800px;
+
```
Comment (LuigiMenez) [inline frontend/src/components/Suggerator/style.jsx]: warning blank line

File: frontend/src/components/CarrousselNews/style.jsx [hunk 3]
```diff
@@ -1,5 +1,17 @@
 import styled from "styled-components";
 
 export default styled.section`
-  height: 20vh;
+height: 70vh;
+
```
Comment (LuigiMenez) [inline frontend/src/components/CarrousselNews/style.jsx]: warning blank line

File: frontend/src/pages/Home.jsx [hunk 4]
```diff
@@ -1,36 +1,3 @@
...
-          href="https://vitejs.dev/guide/features.html"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Vite Docs
-        </a>
-      </p>
-    </header>
-  );
+  return <header className="App-header" />;
```
Comment (LuigiMenez) [inline frontend/src/pages/Home.jsx]: Not Components

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
<header /> in Home is not components .

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not review for the SM

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok

File: frontend/src/components/Header/index.jsx [hunk 5]
```diff
@@ -4,12 +4,7 @@ export default function Header() {
   return (
     <SHeader>
       <h1>Cinépédia</h1>
-      <p>
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Similique
-        repellat mollitia vero sit quod, totam, repudiandae officiis cumque
-        adipisci minus error delectus aperiam labore sint tenetur quam deserunt
-        obcaecati architecto.
-      </p>
+      <p>gdlksfdlskfklfd</p>
```
Comment (LoicBrassart) [inline frontend/src/components/Header/index.jsx:7]: :roll_eyes:

File: frontend/src/components/TypeCarroussel/index.jsx [hunk 6]
```diff
@@ -3,7 +3,8 @@ import STypeCarroussel from "./style";
 export default function TypeCarroussel() {
   return (
     <STypeCarroussel>
-      <p>Ici le carroussel par types de films</p>
+      <h2>By theme</h2>
+      <div>Ici le carroussel par types de films</div>
```
Comment (LoicBrassart) [inline frontend/src/components/TypeCarroussel/index.jsx:7]: Frenglish is not a good idea

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
There are some *very* minor things to change + Be careful to write your comments in English plz

**Pull request merged**